### PR TITLE
Feature/realsoldier

### PR DESCRIPTION
**Pull request by lluciiiia**
1. initializeBoard function 
2. Calculate the coordinate X and Y from the position in the cell in initializePieceImageView function
3. correct the whole board in the right position

### CODE REVIEW

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (lluciiiia) [reply .vscode/settings.json:1]: I removed the file, and it runs!

File: JavaFX/src/main/java/com/example/xiangqi/Controller/InitializeManager.java
[the remark this answers is not in the document]
Comment (lluciiiia) [reply JavaFX/src/main/java/com/example/xiangqi/Controller/InitializeManager.java:1]: removed!

File: JavaFX/src/main/java/com/example/xiangqi/Controller/InitializeManager.java [hunk 1]
```diff
@@ -105,6 +106,16 @@ private void initializePieceImageView() {
                         // Sad case: Check if current clicked is the last piece, if not then remove all previous rectangle
                     });
 
+
+                    //Calculate the coordinate X and Y from the position in the cell
```
[the remark this answers is not in the document]
Comment (lluciiiia) [reply JavaFX/src/main/java/com/example/xiangqi/Controller/InitializeManager.java:110]: removed!

File: JavaFX/src/main/java/com/example/xiangqi/Model/Cell.java [hunk 2]
```diff
@@ -57,9 +55,9 @@ public ImageView getImageView() {
     public void drawRectangle(Cell clickedCell){}
 
     public void drawPieceImageView(ImageView pieceImageView){
-
-        int cellX = this.position[0] * CellConstant.CELL_SIZE; // Calculate the X coordinate (row)
-        int cellY = this.position[1] * CellConstant.CELL_SIZE; // Calculate the Y coordinate (col)
+ 
+        int cellX = (8 - this.position[1]) * 50; // Calculate the X coordinate (row)
```
[the remark this answers is not in the document]
Comment (lluciiiia) [reply JavaFX/src/main/java/com/example/xiangqi/Model/Cell.java:59]: did!

File: JavaFX/src/main/java/com/example/xiangqi/Model/Piece.java [hunk 3]
```diff
@@ -7,64 +7,33 @@
...
+    private PieceName pieceName;
+    private Player player;
 
-	public void setPieceName (PieceName pieceName) {
-		this.pieceName = pieceName;
-	}
+    public Piece(String id, String player, String pieceName) {
+        this.isAlive = true;
+        this.id = id;
+        // Todo: Change the string to player and piece name
```
[the remark this answers is not in the document]
Comment (lluciiiia) [reply JavaFX/src/main/java/com/example/xiangqi/Model/Piece.java:19]: removed!

File: JavaFX/src/main/java/com/example/xiangqi/Model/Cell.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (TranMachSoHan) [reply JavaFX/src/main/java/com/example/xiangqi/Model/Cell.java:59]: Change the 8 and 50 to constant value, you can name the variable to make other people understand what the number does , and put into the CellConstant file

**Pull request merged**